docs/index.rst

```diff
@@ -17,100 +17,157 @@ Installation
 
 $pip install folium
 
+Folium is compatible with Python 2 as well as Python 3.
+
 Base Maps
 ---------
 
-To create a base map, simply pass starting coordinates to Folium, then create the map::
+To create a base map, call on the `Map` class::
 
-    import folium
-    map_osm = folium.Map(location=[45.5236, -122.6750])
-    map_osm.create_map(path='osm.html')
+    Map(location=None, width='100%', height='100%', tiles='OpenStreetMap', API_key=None, max_zoom=18, min_zoom=1, zoom_start=10, attr=None, min_lat=-90, max_lat=90, min_lon=-180, max_lon=180, detect_retina=False)
+
+Parameters:
+
+	location: tuple or list, default None
+	    Latitude and Longitude of Map (Northing, Easting).
+	width: pixel int or percentage string (default: '100%')
+	    Width of the map.
+	height: pixel int or percentage string (default: '100%')
+	    Height of the map.
+	tiles: str, default 'OpenStreetMap'
+	    Map tileset to use. Can use defaults or pass a custom URL.
+	API_key: str, default None
+	    API key for Cloudmade or Mapbox tiles.
+	max_zoom: int, default 18
+	    Maximum zoom depth for the map.
+	zoom_start: int, default 10
+	    Initial zoom level for the map.
+	attr: string, default None
+	    Map tile attribution; only required if passing custom tile URL.
+	detect_retina: bool, default False
+	    If true and user is on a retina display, it will request four
+	    tiles of half the specified size and a bigger zoom level in place
+	    of one to utilize the high resolution.
+
+Returns:
 
-`Live example <http://bl.ocks.org/wrobstory/5545719>`_
+	folium.Map object
+
+
+Example::
+
+    import folium
+    fo_map = folium.Map(location=[45.5236, -122.6750])
 
-Folium defaults to 960 x 500 pixels (to make it easy to generate maps for `bl.ocks <http://bl.ocks.org>`_ ). You can modify the width and height easily::
+You can modify the width and height easily of the map using pixels or percentage::
 
-    map = folium.Map(location=[45.5236, -122.6750],  width=500, height = 300)
+    fo_map = folium.Map(location=[45.5236, -122.6750], width=500, height = 300)
```

Comment:
`fo_`?  I don't like to clobber `map` either, but we can use something more "expressive," like `leaflet_map`, or mimic Basemap and use just `m `.